protocol/contracts/dydx-messages-example/src/contract.rs

```diff
@@ -49,6 +44,38 @@ pub fn execute(
     match msg {
         ExecuteMsg::Approve { quantity } => execute_approve(deps, env, info, quantity),
         ExecuteMsg::Refund {} => execute_refund(deps, env, info),
```

Comment:
nit: I think we can remove these functions, and have this be an example for the new messages only.